presto-testng-services/src/main/java/io/prestosql/testng/services/FlakyTestRetryAnalyzer.java

```diff
@@ -107,6 +98,25 @@ public boolean retry(ITestResult result)
         return true;
     }
 
+    private static boolean isEnabled()
+    {
+        String enabledSystemPropertyValue = System.getProperty(ENABLED_SYSTEM_PROPERTY);
+        if (parseBoolean(enabledSystemPropertyValue)) {
+            return true;
```

Comment:
Preexisting, but this is wrong. when the sys propeerty is "false", we should not be enabled

Reply:
What if sys property is `false` but `CONTINUOUS_INTEGRATION` is set? To me this is `OR` so it is correct as any of this enables the retrying.

Reply:
sys property is "closer to invocation" and thus should override env variable presence

Reply:
Oh... now I get your point. You mean that one should override the other, that makes sense. See: https://github.com/prestosql/presto/pull/6045